src/server/api/controllers/products.controller.js

```diff
@@ -3,22 +3,32 @@ const HttpError = require('../lib/utils/http-error');
 // const moment = require('moment-timezone');
 
 const getProducts = async (req) => {
-  let products = knex('products');
-  const { limit, offset } = req.query;
+  try {
+    let { limit, offset, sortKey, sortOrder } = req.query;
 
-  if (!limit && !offset) {
-    return products;
-  }
+    limit = limit || 10;
 
-  if (limit && offset) {
-    products = products.limit(limit).offset(offset);
-  }
+    offset = offset || 0;
 
-  if (isNaN(limit) || isNaN(offset)) {
-    throw new HttpError('Limit and offset should be a number', 400);
-  }
+    sortKey = sortKey || 'name' || 'price';
```

Comment:
'name' will win if sortKey is not truthy. 'name' is always truthy as it is non empty string which is hard-coded.